Jenkinsfile_nightly

```diff
@@ -0,0 +1,23 @@
+#!groovy
+
+properties([
+        pipelineTriggers([cron('00 21 * * *')]),
```

Comment:
think we should randomise this: https://github.com/hmcts/draft-store/blob/c78daab2621f2a7cad60cd249926ddb4f0f29cd4/Jenkinsfile_nightly#L5